docs/learn/extensions/middleware.md

```diff
@@ -9,7 +9,7 @@ We will be creating an example decorator `with_saving` that saves some metadata
     --8<-- "examples/learn/middleware/with_saving.py:6:7"
 
 
-    --8<-- "examples/learn/middleware/with_saving.py:17:27"
+    --8<-- "examples/learn/middleware/with_saving.py:17:29"
```

Comment:
note to self: double check docs with other change since line highlighting will likely change